### ISSUE TITLE
kernel update

### ISSUE THREAD

**Issue by Magnus-101** [School34erf/GentooInstallScript#1]
Hello, if I need to update the kernel later, what process should I follow? I used the oddlama script but I didn't understand the process, is it the same here? (im noob)

**Comment by School34erf** [comment]
Yup its the same here

**Comment by Magnus-101** [comment]
Ok  ;)

**Comment by School34erf** [comment]
I don't think that it should create any problems with the script and updating the kernel

**Comment by School34erf** [comment]
Just update it like you usually do
